airflow/www/compile_assets.sh

```diff
@@ -31,4 +31,9 @@ fi
 yarn install --frozen-lockfile
 yarn run build
 
-find package.json yarn.lock static/css static/js -type f | sort | xargs md5sum > "${MD5SUM_FILE}"
+YARN_LOCK_FILE="yarn.lock"
+if [[ -f "$YARN_LOCK_FILE" ]]; then
```

Comment:
We shouldn't be running this script if there is no lock file -- else we have run `yarn install --frozen-lockfile` which will (I hope!) fail.

Reply:
Yep. This was what I fixed in my PR.

Reply:
the whole script is not run when we install from packages.

Reply:
updated in https://github.com/apache/airflow/pull/16577/commits/11c927fc07a53f2a9334121f06e4426c8bb438e0